go.mod

```diff
@@ -146,7 +146,7 @@ require (
 	github.com/mdlayher/netlink v1.7.2
 	github.com/microsoft/go-mssqldb v1.7.2 // replaced
 	github.com/microsoft/kiota-authentication-azure-go v1.0.2
-	github.com/microsoftgraph/msgraph-sdk-go v1.43.0
+	github.com/microsoftgraph/msgraph-sdk-go v1.46.0
```

Comment:
Thanks. I noticed msgraph-sdk-go was weird in our last update batch and kept it at v1.43.0 (https://github.com/gravitational/teleport/pull/43828#issuecomment-2207395879), but apparently some of the indirects in other modules skipped. I figured a fix would land at some point.